python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -18,9 +18,12 @@
 from .GangaRepository import SchemaVersionError
 
 import xml.sax.saxutils
+import copy
 
 logger = getLogger()
 
+_eval_strings = {}
```

Comment:
We should avoid globals. This probably makes most sense as a class attribute of `Loader`. Also, perhaps rename to `_eval_string_cache` or similar?